### ISSUE TITLE
Long group names are hidden when they wrap onto a second line

### ISSUE THREAD

**Issue by bree-z** [cuny-academic-commons/commons-in-a-box#350]
This issue was reported in the [CBOX OpenLab support forum](https://commonsinabox.org/groups/openlab-help-support/forum/topic/long-group-names-cut-off/).

When a long group name wraps onto a second line, the text overflow is hidden.

Here's the example from the BMCC's OpenLab: https://openlab.bmcc.cuny.edu/groups/ece-110-lecture-longley/

I reproduced the issue here: https://openlabdev.commonsinabox.org/groups/new-course-2-winter-2021/

And here's what it looks like on openlabdev.org: http://openlabdev.org/groups/last-clone-for-today/

Thanks!

**Comment by boonebgorges** [comment]
I've made an adjustment so that the title area will expand when the title breaks onto multiple lines. This change rolls back some absolute positioning that's been in place forever on the OpenLab, so when testing I recommend looking at different kids of object types (users, group; mobile, desktop) to make sure things don't break unexpectedly.

On mobile, I had to choose what to do with the hamburger button. It looks better aligned with the horizontal rule under the title, but this has the potential to break the navigation if the title is very long, so I opted instead to keep it at the top. See screenshot.

![Screenshot_2021-06-03_14-37-38](https://user-images.githubusercontent.com/246627/120702368-9da73500-c479-11eb-8b79-0d861b33ae23.png)
